src/status_im/utils/security.cljs

```diff
@@ -24,7 +24,7 @@
     data))
 
 ;; Links starting with javascript:// should not be handled at all
-(def javascript-link-regex #"javascript://.*")
+(def javascript-link-regex #"(?i)javascript://.*")
```

Comment:
if we aren't rendering anything but `http` and `https` as clickable, are these needed?

Reply:
technically no, but given that status-go is a separate component, if there's any regression there, then status-react would blindly follow those links. This code is only run once the user clicks on a link, so performance is not impacted, therefore I would probably keep it, just to be on the safe side. what do you think?

Reply:
I don't mind having multiple security checks like this.. i'd rather over validate than under.